common/navbar.js

```diff
@@ -365,6 +365,7 @@
                     //  - if resolverImplicitCatalog === catalogId:   /id/RID
                     //  - otherwise:                                  /id/catalogId/RID
                     scope.ridSearch = function () {
+                        $rootScope.showSpinner = true;
```

Comment:
Since this will not change the current page location (and will open a tab instead), I think we shouldn't show the spinner that we show for the whole page. I think instead we should just disable the current input and change the icon of it to show spinner. This is how I did it for export and share buttons.

Reply:
Sounds good. That's better in this case